hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/keys/GetKeyHandler.java

```diff
@@ -80,7 +88,7 @@ protected void execute(OzoneClient client, OzoneAddress address)
       IOUtils.copyBytes(input, output, chunkSize);
     }
 
-    if (isVerbose()) {
+    if (isVerbose() && !"/dev/null".equals(dataFile.getAbsolutePath())) {
```

Comment:
Hardcoded path like "/dev/null" may not work in other platforms like Windows.

Reply:
I think that's OK.    This check does not affect functionality and we're not relying on any specific property of `/dev/null` here.  The worst thing that can happen is that if someone issues `ozone sh --verbose key get ... /dev/null` on Windows, hash is not printed.  But I doubt that's a valid file on Windows anyway.

Reply:
Sounds good to me. +1